src/actions/actionTypes.js

```diff
@@ -6,6 +6,9 @@ export const PASSWORD_RESET= "PASSWORD_RESET";
 export const NEW_PASSWORD="NEW_PASSWORD";
 export const USER_NOT_FOUND="USER_NOT_FOUND";
 export const REGISTER_USER= "REGISTER_USER";
+export const LIKE = "LIKE";
+export const DISLIKE = "DISLIKE";
+
 const ACTION_TYPE = {
   NEW_ARTICLE: "NEW_ARTICLE",
   VIEW_ARTICLES: "VIEW_ARTICLES",
```

Comment:
Add your actions to the ACTION_TYPE dictionary, and use the exported ACTION_TYPE to call your actions, so as to have code consistency

Reply:
@LehruAngela i have made this change